docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -183,32 +174,33 @@ Modules & Plugins
 .. _coll_docs_structure_reqs:
 
 Documentation requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All modules and plugins MUST:
+Collections:
 
-* Include a :ref:`DOCUMENTATION <documentation-block>` block.
-* Include an :ref:`EXAMPLES <examples-block>` block (except where not relevant for the plugin type).
-* Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection (including ``ansible.builtin`` for the listed entities from ansible-core.
-  
-When using ``version_added`` in the documentation:
+* ``MUST`` use :ref:`links and formatting macros <linking-and-other-format-macros-within-module-documentation>`.
+* ``SHOULD`` have contributor guidelines in the ``CONTRIBUTING.md`` or ``README.md`` file.
 
-* Declare the version of the collection in which the options were added -- NOT the version of Ansible.
-* If you for some reason really have to specify version numbers of Ansible or of another collection, you also have to provide ``version_added_collection: collection_name``. We strongly recommend to NOT do this.
-* Include ``version_added`` when you add new content (modules, plugins, options) to an existing collection. The values are shown in the documentation and can be useful, but you do not need to add ``version_added`` to every option, module, and plugin when creating a new collection.
+All modules and plugins:
 
-Other items:
+* ``MUST`` include a :ref:`DOCUMENTATION <documentation-block>` block.
+* ``MUST`` include an :ref:`EXAMPLES <examples-block>` block (except where not relevant for the plugin type).
+* ``MUST`` Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection including ``ansible.builtin.`` for ansible-core.
+* ``MUST`` include a :ref:`RETURN <return-block>` block for modules and other plugins that return data.
+* ``MUST`` include the ``version_added`` field when adding new content to an existing collection for entities that support it, for example, for modules, plugins, options, return values, and attributes.
 
-* The ``CONTRIBUTING.md`` (or ``README.md``) file MUST state what types of contributions (pull requests, feature requests, and so on) are accepted and any relevant contributor guidance. Issues (bugs and feature requests) reports must always be accepted.
-* Collections are encouraged to use :ref:`links and formatting macros <linking-and-other-format-macros-within-module-documentation>`
-* Including a :ref:`RETURN <return-block>` block for modules is strongly encouraged but not required.
+  * You do not have to add ``version_added`` when creating a new collection before its first release.
+  * Declare the version of the collection in which the options were added -- ``NOT`` the version of Ansible/ansible-core.
+  
+    * If you for some reason really have to specify version numbers of Ansible or of another collection, you also ``MUST`` to provide ``version_added_collection: collection_name``. We strongly recommend to ``NOT`` do this.
```

Comment:
```suggestion
    * If, for some reason, you need to specify version numbers of Ansible or another collection, you ``MUST`` also provide ``version_added_collection: collection_name``. We strongly recommend to ``NOT`` do this.
```

Reply:
Since this is something we strongly recommend against, can we add some justification for that? Does ``version_added_collection: collection_name`` result in longer term maintenance headaches? Does it lead to user confusion?

Reply:
Good catch. I don't remember the context of this and can't come up with possible scenarios. Maybe it was relevant to the time right after the ansible/ansible split and could be removed now.
@felixfontein maybe you remember why someone might want to do it?